### PR TITLE
Add More Front Matter

### PR DESCRIPTION
**Pull request by blink1073**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,84 @@
...
+
+Follow the source build instructions first.
+
+```bash
+cd example
+npm install
+npm run build
+```
+
+Start a web server in the `example` folder and navigate to the served URL.
```
Comment (sccolbert) [inline README.md]: This example actually runs without a web server.

**Comment by blink1073** [comment]
This is ready.

**Comment by blink1073** [comment]
Or not, merge conflicts.

File: src/index.ts [hunk 2]
```diff
@@ -102,6 +102,7 @@ const SHORTCUT_CLASS = 'p-CommandPalette-shortcut';
  */
 const TITLE_CLASS = 'p-CommandPalette-title';
 
+const FOO = 'bar';
```
Comment (sccolbert) [inline src/index.ts]: ??

**Pull request merged**